### PR TITLE
Initial support for CA Certificates for endpoints

### PR DESCRIPTION
**Pull request by bmo-at-a9s**
This PR adapts https://github.com/cloudfoundry/stratos/pull/4824 into a state that may be merged into our repo.
For now, it is in draft stage. It should NOT be merged unless taken out of draft stage.

### CODE REVIEW

**Comment by bmo-at-a9s** [comment]
Before this gets merged, I would like to wait for work on https://github.com/anynines/stratos/tree/feature/INT-2910-GH-Actions-cf-deploy to finish and be merged back into `develop`, so as to be able to rebase this branch on those changes and make sure everything still works.

**Comment by bmo-at-a9s** [comment]
@mafolz

**Comment by bmo-at-a9s** [comment]
This can now be reviewed and is no longer a draft, but merging should stay blocked by https://github.com/anynines/stratos/tree/feature/INT-2910-GH-Actions-cf-deploy

**Review by mafolz — APPROVED** [review]

**Review by mafolz — CHANGES_REQUESTED** [review]

File: src/frontend/packages/core/src/features/endpoints/create-endpoint/create-endpoint-cf-step-1/create-endpoint-cf-step-1.component.html
Comment (bmo-at-a9s) [reply src/frontend/packages/core/src/features/endpoints/create-endpoint/create-endpoint-cf-step-1/create-endpoint-cf-step-1.component.html]: Done 👌🏻

File: src/jetstream/http_client.go
Comment (bmo-at-a9s) [reply src/jetstream/http_client.go]: Done 👌🏻

File: src/jetstream/http_client.go
Comment (bmo-at-a9s) [reply src/jetstream/http_client.go]: fixed

**Review by mafolz — APPROVED** [review]

**Pull request merged**
